datadog_checks_base/datadog_checks/base/utils/http.py

```diff
@@ -827,6 +827,11 @@ def __init__(self, config):
             self._fetch_options['client_id'] = self._client_id
             self._fetch_options['client_secret'] = self._client_secret
 
+        self._auth_params = config.get('auth_params', {})
```

Comment:
Name suggestions:

- oauth_params
- oauth_options
- oauth_headers (that's _usually_ what they end up as, but I guess not always)

I think having "oauth" in the name highlights that this won't (shouldn't) work with other mechanisms.

Reply:
`auth_params` is already nested in the oauth type

Reply:
Hmm, I guess we'll be adding docs for this in [here](https://github.com/DataDog/integrations-core/blob/e4509fd3d3946d0622e8be749aae3a6a8229cd4f/datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/http.yaml#L190) right?

I think you're right, and that's enough mention of `oauth` and in fact `auth` too :)

Then I vote to just say `params` or `options`. "auth" is redundant in this context.

Reply:
> Hmm, I guess we'll be adding docs for this in [here](https://github.com/DataDog/integrations-core/blob/e4509fd3d3946d0622e8be749aae3a6a8229cd4f/datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/http.yaml#L190) right?

Yep, I'll add that in a follow-up PR

> Then I vote to just say params or options. "auth" is redundant in this context.

Okay! @tuteng Could you rename it to `options`? Once done we'll be able to merge this one!

Reply:
Fixed, thanks for everyone comment, PTAL